app/javascript/utils/Sorting.js

```diff
@@ -0,0 +1,40 @@
+/**
+  * Compare function for sorting the Object.entries elements in the jurisdiction drop down list
```

Comment:
Idk if i like this comment explaining what the function does...They're not Object entries, they're jurisdiction strings (which happen to be the keys).